extensions/blocks/map/map.php

```diff
@@ -40,6 +40,22 @@ function jetpack_get_mapbox_api_key() {
 	return Jetpack_Options::get_option( 'mapbox_api_key' );
 }
 
+/**
+ * Record a Tracks event every time the Map block is loaded on WordPress.com and Atomic.
+ */
+function jetpack_record_mapbox_load_event() {
+	$event_name = 'jetpack_map_block_mapbox_load';
+	$user       = wp_get_current_user();
+	if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+		require_lib( 'tracks/client' );
+		tracks_record_event( $user, $event_name, array( 'site_id' => get_current_blog_id() ) );
+		return;
+	} elseif ( jetpack_is_atomic_site() ) {
```

Comment:
I'm not opposed to the change obviously, but I'd love to have more context on this.
As far as I can tell, `is_active` checks if JP is connected to WPCOM.
So, is it possible for an Atomic site to disconnect from JP?

Or maybe you mean that `is_active` also checks other stuff (e.g. if a site has accepted the ToS). 🤔

Reply:
> As far as I can tell, is_active checks if JP is connected to WPCOM.

That's correct.

> is it possible for an Atomic site to disconnect from JP?

Normally not, but there are some use-cases where the connection breaks.

It may be enough of an edge-case for us to ignore, though. I am not insisting on it.